test/AspNet.Security.OAuth.Providers.Tests/Zoho/bundle.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://raw.githubusercontent.com/justeat/httpclient-interception/master/src/HttpClientInterception/Bundles/http-request-bundle-schema.json",
+  "items": [
+    {
+      "uri": "https://accounts.zoho.com/oauth/v2/token",
+      "method": "POST",
+      "contentFormat": "json",
+      "contentJson": {
+        "access_token": "secret-access-token",
+        "token_type": "Bearer",
+        "refresh_token": "secret-refresh-token",
+        "expires_in": 3600
+      }
+    },
+    {
+      "uri": "https://accounts.zoho.com/oauth/user/info",
+      "method": "GET",
+      "contentFormat": "json",
+      "contentJson": {
+        "ZUID": "1234567890",
+        "Email": "testuser@example.com",
+        "Display_Name": "User Name"
+      }
+    },
+    {
+      "uri": "https://accounts.zoho.com/oauth/serverinfo",
+      "method": "GET",
+      "contentFormat": "json",
+      "contentJson": {
+        "result": "success",
+        "locations": {
+          "eu": "https://accounts.zoho.eu",
+          "au": "https://accounts.zoho.com.au",
+          "in": "https://accounts.zoho.in",
+          "jp": "https://accounts.zoho.jp",
+          "uk": "https://accounts.zoho.uk",
+          "us": "https://accounts.zoho.com",
+          "ca": "https://accounts.zohocloud.ca",
+          "sa": "https://accounts.zoho.sa"
+        }
+      }
+    }
```

Comment:
Is this used by anything?

Reply:
No, it was used in a previous version of the Zoho implementation. I will remove it.